charts/longhorn/v1.1.3/README.md

```diff
@@ -22,11 +22,41 @@ Longhorn is 100% open source software. Project source code is spread across a nu
 3. Make sure `bash`, `curl`, `findmnt`, `grep`, `awk` and `blkid` has been installed in all nodes of the Kubernetes cluster.
 4. Make sure `open-iscsi` has been installed, and the `iscsid` daemon is running on all nodes of the Kubernetes cluster. For GKE, recommended Ubuntu as guest OS image since it contains `open-iscsi` already.
 
+## Installation
```

Comment:
@innobead I don't see these README.md changes on https://github.com/rancher/charts/pull/1639#pullrequestreview-834610717 or https://github.com/rancher/charts/pull/1660#pullrequestreview-834605402.

Was this intentional?

Reply:
1.1.3 is a new release folder.

Reply:
Yes, but what I mean is that it seems like we're updating the README.md within 1.1.3 in this branch but not on dev-v2.5 / dev-v2.6.

e.g. now that https://github.com/rancher/charts/pull/1660 is merged, if you look at the README.md of the merged longhorn `100.0.1+up1.1.3` chart [here](https://github.com/rancher/charts/blob/dev-v2.6/charts/longhorn-1.1/longhorn/100.0.1%2Bup1.1.3/README.md), it does not show these additional installation steps that we added here to this version of the 1.1.3 chart.

Should dev-v2.5 and dev-v2.6 also be updated accordingly before release?